ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1768,12 +1754,6 @@ void ConfigFdbTableEntry(struct mac_learning_info learn_info,
   return;
 }
 
-void ConfigIpTunnelTermTableEntry(struct tunnel_info tunnel_info,
```

Comment:
Please remove the declaration from ovs-p4rt.h if this is not used anymore

Reply:
Yes, this declaration was removed.
https://github.com/ipdk-io/ovs/commit/36a2b33763b17ba092cab42847ef92888f50eb37#diff-b609589c1ca4f1d69dda477121067a8e5e17564ada29fa90d912613fb23010aaL97